### ISSUE TITLE
Weigh multiple losses

### ISSUE THREAD

**Issue by lhk** [fchollet/keras#4727]
I've built a model with multiple outputs and need to balance the losses against each other.
The theano backend can do this via the `loss_weights `parameter in `model.compile()`.
Unfortunately I'm using tensorflow.

How does Keras deal with multiple losses:
Do you add them and then take the derivative of the sum ?
Or do you optimize them individually ?

My idea is to just implement the losses I need myself and multiply the weighting factor into the loss manually. But that would only work if you're optimizing the sum of losses.

**Comment by fchollet** [comment]
> The theano backend can do this via the loss_weights parameter in model.compile().
> Unfortunately I'm using tensorflow.

The API is the exact same whether you are using TF or Theano.

**Comment by lhk** [comment]
Well then the docs are wrong

> kwargs: when using the Theano backend, these arguments are passed into K.function. Ignored for Tensorflow backend.

**Comment by lhk** [comment]
Even comments in the actual source code say that tensorflow ignores this.

**Comment by pGit1** [comment]
ping @fchollet . Is he correct? 

@lhk what exactly does this loss_weights parameter actually do?
